packages/babel-plugin-minify-infinity/src/index.js

```diff
@@ -6,6 +6,12 @@ module.exports = function({ types: t }) {
     t.numericLiteral(1),
     t.numericLiteral(0)
   );
+  const badTransforms = {
+    ArrayPattern: true,
+    AssignmentExpression: true,
```

Comment:
Doesn't this prevent transforming `x = Infinity`?

Reply:
Yes, I missed testing that case.